cpp/lib/include/FileUtil.h

```diff
@@ -66,8 +66,8 @@ class ReadLines {
     public:
         std::string operator*();
         void operator++();
-        bool operator==(const const_iterator &rhs);
-        inline bool operator!=(const const_iterator &rhs) { return not operator==(rhs); }
+        bool operator==(const const_iterator &rhs) const;
```

Comment:
These const changes should have been part of my pull request this morning already (FileUtil + MBox)